socs/agents/acu/agent.py

```diff
@@ -111,6 +116,9 @@
     ('ACU_corotator', 'corotator', None, None),
     ('ACU_shutter', 'shutter', None, None),
     ('ACU_tilt', 'tilt_slow', 'changed', 0.5),
+    (None, 'tilt_fast', None, None),
```

Comment:
This (having `None` for the `block_name`) just means the `tilt_fast` fields aren't being published to a feed, right?

Reply:
Yes... these numbers turned out to be useless.